internal/server/target.go

```diff
@@ -15,6 +15,10 @@ import (
 	"time"
 )
 
+const (
+	HTTP_STATUS_CLIENT_DISCONNECTED = 499
```

Comment:
A small nit, and I appreciate it's too late for a review since the changes landed already.

Outside of the Go standard library, where all-caps constants are often used for historic or compatibility reasons (e.g., POSIX permissions constant, syscall flags, etc.), `HTTP_STATUS_CLIENT_DISCONNECTED` could be made more Go-ish as `StatusClientClosedRequest` (or even starting with lower-case to avoid exporting the symbol if there is no need to reuse it elsewhere).

Eventually, some linters or other Go developer might pick on this, too. :smile: Per the:

- [Effective Go / MixedCaps](https://go.dev/doc/effective_go#mixed-caps)

Why the name change of the variable? To keep it more aligned with what Nginx initially called it. This might have become a canon now since others (i.e., Cloudflare) picked it, too. This is also a custom status code introduced by Nginx; it does not exist (to best of my knowledge) in the original RFC, etc.

Reply:
D'oh, thanks @kwilczynski! Despite writing Go for years I still accidentally write an all-caps constant from time to time. Old habits, and all that :) Thanks for pointing it out.

Good call on the name change too. I'll take care of it.